src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -315,7 +317,7 @@ const SearchFieldList = ({ onSearch }: Props) => {
   );
 
   return (
-    <Card style={{ padding: '1rem' }}>
+    <Card style={{ padding: '1rem' }} className={styles.people_section}>
```

Comment:
Maybe consider making this a different CSS class to add the padding.  That might be unnecessary, but you could accomplish this by making a people_section_alt using the &_alt notation in SCSS.